packages/mask/src/plugins/MaskBox/apis/merkleProof.ts

```diff
@@ -0,0 +1,16 @@
+import urlcat from 'urlcat'
+import { MERKLE_PROOF_ENDPOINT } from '../constants'
+
+export async function getMerkleProof(leaf: string, root: string) {
+    try {
+        const response = await fetch(
+            urlcat(MERKLE_PROOF_ENDPOINT, {
+                leaf,
+                root: root.replace(/^0x/, ''),
+            }),
+        )
+        return (await response.json()) as { proof?: string[]; message?: string; module?: string }
+    } catch (err: any) {
+        throw new Error(err?.message)
+    }
+}
```

Comment:
```suggestion
    } catch (error: unknown) {
        throw error
    }
}
```

Reply:
then why you need to add a try-catch anyway?